src/hooks/useHandleFollowChannel.tsx

```diff
@@ -1,27 +1,58 @@
+import React from 'react'
+
 import { useFollowChannel, useUnfollowChannel } from '@/api/hooks'
+import { useDialog } from '@/providers/dialogs'
 import { usePersonalDataStore } from '@/providers/personalData'
+import { Text } from '@/shared/components/Text'
 import { SentryLogger } from '@/utils/logs'
+import { UnfollowDescriptionContainer } from '@/views/viewer/ChannelView/ChannelView.style'
```

Comment:
This import shouldn't be here, this component should be declared next to the hook instead